<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_dialog</class>
 <widget class="QDialog" name="login_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>68</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_line_edit">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>60</y>
     <width>222</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line_edit">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>110</y>
     <width>222</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>118</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QComboBox" name="color_combo_box">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>160</y>
     <width>222</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../ressource/game_piece_red.png</normalon>
     </iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ressource/game_piece_blue.png</normaloff>../ressource/game_piece_blue.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ressource/game_piece_green.png</normaloff>../ressource/game_piece_green.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ressource/game_piece_yellow.png</normaloff>../ressource/game_piece_yellow.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
